shared/src/main/java/org/odk/collect/shared/livedata/LiveDataUtils.java

```diff
@@ -0,0 +1,113 @@
+package org.odk.collect.shared.livedata;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MediatorLiveData;
+
+import kotlin.Triple;
+
+public class LiveDataUtils {
+
+    private LiveDataUtils() {
+
+    }
+
+    public static <T, U, V> LiveData<Triple<T, U, V>> zip3(LiveData<T> one, LiveData<U> two, LiveData<V> three) {
```

Comment:
None of the `zip3` components are in use, right? I'd remove them and re-add if needed.

Reply:
Will do